docs/pages/application-access/okta/guide.mdx

```diff
@@ -0,0 +1,169 @@
+...
+[
+  {
+    "id": "111111",
+    "label": "Okta Admin Console"
+  },
+  {
+    "id": "222222",
+    "label": "Okta Browser Plugin"
+  },
+  {
+    "id": "333333",
+    "label": "Okta Dashboard"
+  },
+  {
+    "id": "444444",
+    "label": "SAML test"
+  },
+  ...
+]
+```
+
+From here, you can copy the IDs from the output of the `curl` commands and use
+these to create the Okta import rule. Put this in a file called `okta-import-rule.yaml`:
+
+```yaml
+kind: okta_import_rule
+version: v1
+metadata:
+  name: import-rule
+  description: "Okta import rule for all groups and applications"
+spec:
+  mappings:
+  - match:
+    - app_ids: ["111111", "222222", "333333"] # Replace these with your Okta application IDs.
+    add_labels:
+      app_label: app-label-value
+  - match:
+    - group_ids: ["111111", "222222", "333333"] # Replace these with your Okta group IDs.
+    add_labels:
+      group_label: group-label-value
+```
+
+This will ensure that the applications and groups, once synchronized to Teleport,
+will be given the specified labels.
+
+## Step 2/3. Set up the Okta service.
+
+In this step, we'll set up the Okta service.
+
+### Get a join token
+
+Establish trust between your Teleport cluster and the new Okta Service
+instance by creating a join token:
+
+```code
+$ tctl tokens add --type=okta --ttl=1h --format=text
+(=presets.tokens.first=)
+```
+
+On the host where you will install the Teleport Okta Service, create a
+file called `/tmp/token` that consists only of your token:
+
+```code
+$ echo <Var name="join-token" /> | sudo tee /tmp/token
+```
+
+### Configure the Okta service
+
+First, on the host where you will run the Okta service , create a file at
+<Var name="okta-api-token-path" /> with the contents of the API token.
+
+Next, on the same host, create a file at `/etc/teleport.yaml` with the
+following content:
+
+```yaml
+version: v3
+teleport:
+  join_params:
+    token_name: "/tmp/token"
+    method: token
+  proxy_server: "teleport.example.com:443"
+auth_service:
+  enabled: no
+proxy_service:
+  enabled: no
+ssh_service:
+  enabled: no
+okta_service:
+  enabled: yes
+  api_endpoint: <Var name="okta-endpoint-url" />
+  api_token_path: <Var name="okta-api-token-path" />
+```
+
+Edit `/etc/teleport.yaml` to replace `teleport.example.com:443` with the host
+and port of your Teleport Proxy Service or Teleport Cloud tenant, e.g.,
+`mytenant.teleport.sh:443`.
+
+### Run the Teleport Application Service
+
+(!docs/pages/includes/start-teleport.mdx service="the Teleport Application Service"!)
+
+## Step 3/3. Connect to the Teleport web UI
+
+Navigate to the application access page within the Teleport web UI to see the list of
+of synchronized Okta applications. If you're unable to see the applications, please refer
+to the application [Access Controls](../controls.mdx) and update your roles to match the
+labels given to your applications by the above specified Okta import rules.
```

Comment:
I read all the way to the end, and even if I've followed the steps I still don't quite understand what I've done or why I would have wanted to do it to begin with.

I think we need to reframe this guide to be more about the task a user can accomplish with the Okta service.